force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +469,13 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+        if(person1.getAge() > person2.getAge()){
+            return person1;
+        } else if(person1.getAge() < person2.getAge()){
+            return person2;
+        }else{
+            return person1;
+        }    
+ // Replace null with the variable you used to store the result
```

Comment:
To return person1 if their age is greater or equal to person2, you can combine some of your statements:
```suggestion
        if(person1.getAge() >= person2.getAge()){
            return person1;
        } else {
            return person2;
        }
 // Replace null with the variable you used to store the result
```

When you learn and feel more comfortable with the **[ternary operator](https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/langCon_apex_expressions_operators_understanding.htm)**, you can shorten it to the following:
```
 public static Person getOlderPerson(Person person1, Person person2) {
        return person1.age >= person2.age ? person1 : person2;
    }
```